spring-cloud-task-core/src/main/java/org/springframework/cloud/task/repository/support/SimpleTaskRepository.java

```diff
@@ -59,7 +59,7 @@ public TaskExecution completeTaskExecution(long executionId, Integer exitCode, D
 		validateExitInformation(executionId, exitCode, endTime);
 		exitMessage = trimExitMessage(exitMessage);
 		taskExecutionDao.completeTaskExecution(executionId, exitCode, endTime, exitMessage);
-		logger.debug("Updating: TaskExecution with executionId="+executionId
+		logger.info("Updating: TaskExecution with executionId="+executionId
```

Comment:
I thought this was to be debug to prevent unwanted data from appearing in the log.